build.gradle

```diff
@@ -1,27 +1,16 @@
-apply plugin: 'com.github.ben-manes.versions'
-
-buildscript {
-  apply from: 'gradle/dependencies.gradle'
-
-  repositories {
-    mavenCentral()
-    google()
-    gradlePluginPortal()
-    //mavenLocal()
-  }
-
-  dependencies {
-    classpath deps.plugins.kotlin
-    classpath deps.plugins.android
-    classpath 'com.vanniktech:gradle-maven-publish-plugin:0.20.0'
-    classpath 'org.ajoberstar.grgit:grgit-gradle:5.0.0'
-    classpath 'org.jetbrains.dokka:dokka-gradle-plugin:1.6.21'
-    classpath deps.plugins.versions
-    classpath deps.plugins.spotless
-    classpath "app.cash.paparazzi:paparazzi-gradle-plugin:${versions.releasedPaparazzi}"
-  }
+plugins {
```

Comment:
Thanks for contributing!

I'd prefer to keep using the `buildscript {}` here.

Also, can we removing the `[plugins]` section in the toml, and add a `plugins-` suffix to keep the `plugins.kotlin` notation as above.

Reply:
Reverted back to buildscript.